code/datums/mood.dm

```diff
@@ -73,6 +73,11 @@
 	return ..()
 
 /datum/mood/process(seconds_per_tick)
+	if(!mob_parent)
```

Comment:
This is just papering over the issue of either, map_parent ceasing to exist without cleaning up the mood datum, or a mood datum being created without a mob_parent. The core issue should be fixed.

Reply:
I am unable to replicate what is causing the core issue.  However this solution prevents a lot of runtime errors from spamming the logs. Instead it creates one stack trace which is helpful.

Reply:
Get this testmerged if you'd like to use this to try to find the issue, but this is not a proper fix. Reducing runtimes in the log should not be the end goal.